nilearn/reporting/utils.py

```diff
@@ -41,7 +45,7 @@ def _figure_to_svg_bytes(fig):
 
 def _figure_to_png_bytes(fig):
     """Save figure as png and return it as bytes."""
-    return _figure_to_bytes(fig, format="png")
+    return _figure_to_bytes(fig, format="png", bbox_inches="tight")
```

Comment:
to monitor in the future but ensured that labels were not cut out of the figures 

see https://github.com/nilearn/nilearn/pull/5202#issuecomment-2718642623